src/components/metadata/field.js

```diff
@@ -174,7 +192,50 @@ class StaticField extends React.PureComponent {
 }
 
 
-module.exports = {
-  MetadataField,
-  StaticField
+
+
+const DragSourceSpec = {
+  beginDrag({ property, isMixed, itemsSelected, text  }) {
+    console.log('DRAG', property.id)
+    return {
```

Comment:
For the drag item I think we will want to add the following properties: `isMixed`, `text` and `type` (the drag preview will have to format the text accordingly; alternatively we could do it here, and pass in something like `value: auto(text, type)`), `from` (which should be the `property.id`) and instead of `itemsSelected` we can set an `id` which should be an array of all selected subject ids.

Instead of passing down all the items we'll pass down an `onDrag` handler. The handler will be defined up in the `MetadataPanel` and we'll pass it everything except for the item ids. So something like this:

```javascript
beginDrag({ property, isMixed, text, type, onDragStart }) {
  return onDragStart({
    from: property.id,
    value: auto(text, type)
  })
}
```

In the `MetadataPanel` we'll have `handleDragItemField` which adds the `id` array. (And similarly `handleDragSelectionField` and `handleDragPhotoField`).

Reply:
Actually, I just realized we'll probably want to push down the 'subjects' after all: we also need to make sure that you cannot drag and drop e.g. an item field onto a photo field (this does not make sense because we're moving fields within the same subject, respectively). So we also need the ids for a `canDrop` check. Because of that let's just push down the subjects too.